docs/src/components/DocsPageBuilder/ComponentScreenshots.js

```diff
@@ -56,9 +56,11 @@ const ComponentScreenshot = ({ title, subText, ...rest }) => {
       <BpkDescriptionTerm
         className={getClassName('bpkdocs-component-screenshots__item-title')}
       >
-        <BpkText tagName="h3" textStyle="xs">
-          {title}
-        </BpkText>
+        {title && (
+          <BpkText tagName="h3" textStyle="xs">
+            {title}
+          </BpkText>
+        )}
```

Comment:
Made title optional to make some proptype errors we were seeing in the console go away.